src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -97,7 +100,7 @@ public virtual void InstallPlatformSpecificSkeletonDependencies(StringBuilder st
             var snippet = new StringBuilder();
             snippet
                 .AppendLine()
-                .AppendLine($"if grep -q cli \"/opt/oryx/.imagetype\"; then")
+                .AppendLine($"if grep -q -e cli \"/opt/oryx/.imagetype\"; then")
```

Comment:
We need to add -e githubactions \"/opt/oryx/.imagetype\ if we are enabling dynamic installation for github actions images as well. If we are not, we can remove -e. Essentially, -e option is used if we need to match multiple patterns. -e is added for each pattern.

Reply:
yah i don't think we want to dynamicall install packages for github actions as well, so removing `-e` should be the way to go here

Reply:
Good catch! I've removed `-e`